azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -113,10 +113,13 @@ def create_from_edge_environment(cls):
         """
         Instantiate the client from the IoT Edge environment.
 
-        This method can only be run from inside an IoT Edge container.
+        This method can only be run from inside an IoT Edge container, or in a debugging
+        environment configured for Edge development (e.g. Visual Studio, Visual Studio Code)
```

Comment:
>Visual Studio,  [](start = 58, length = 15)

not sure if Visual Studio has an extension - i think it's only VS Code but i might be wrong.

Reply:
I'm pretty sure it's both.

---
In reply to: [297895299](https://github.com/Azure/azure-iot-sdk-python-preview/pull/117#discussion_r297895299) [](ancestors = 297895299)